apps/hashdotai/blog/000_what-is-agent-based-modeling.mdx

```diff
@@ -0,0 +1,55 @@
+
+## What is a model?
+
+Models attempt to represent and simulate real or imagined scenarios, which we call the ‘target’.
+
+Models are great because they force us to consider the many features of a target, and express the relationships between them. This in turn [allows for discovery](<http://onlinelibrary.wiley.com/doi/10.1002/(SICI)1520-6858(1999)2:2%3C69::AID-SYS3%3E3.0.CO;2-0/abstract>) of potential emergent phenomenon.
+
+Nigel Gilbert’s [little green book](http://methods.sagepub.com/book/agent-based-models) on agent-based models considers three primary types of models:
+
+1. **Scale models** are those which attempt to replicate targets at a smaller-scale. System detail or complexity is reduced accordingly. For example, a scale model of an apartment complex might show room shapes and doorways, but might not include such intricate detail as locks on the doors, or light switches on the walls.
+1. **Ideal-type models** simplify problems by exaggerating some characteristic of a target (e.g. assuming an economy is [perfectly competitive](https://en.wikipedia.org/wiki/Perfect_competition), or that information [flows instantly](https://en.wikipedia.org/wiki/Perfect_information) between all participants in a market, without any misunderstandings).
+1. **Analogical models**, most often applied to systems not yet well understood, draw on well-understood real-world phenomena to speculate how a system might work. These analogical models are rarely perfect, but often an constitute important first step in tackling a problem. For example, in 1808 Dalton sought to explain atomic theory using a [Billiard Ball Model](https://edtech2.boisestate.edu/lindabennett1/502/dalton.html). This was later replaced by the Cubic Model, the [Plum Pudding Model](https://link.springer.com/chapter/10.1007/978-3-540-70626-7_9), the Saturnian model, the Rutherford model, the Bohr Model, and the Atomic Orbital model, each in turn).
+
+Agent-based models, or ABM, are sometimes referred to as ABMS. This term denotes ‘agent-based bodeling and **simulation**’ more fully.
+
+## So what is an *agent-based* model?
+
+An agent-based model consists of 4 core components.
+
+Key components of an Agent-Based Model:
+
+1. **Agents:** social things in a model that can interact with each other and an environment, and pass information between each other. Agents might represent animals, individuals, households, organizations, or even entire countries.
+1. **Properties:** agents have properties. A property might be memory; a state, characteristic, or attribute, such as hunger, speed, or health. Properties are discrete, and can be binary (yes/no), or numerical (e.g. on a 1–100 scale).
```

Comment:
Is 'memory' part of this list? That's what I've assumed here. I also assume 'binary' and 'numerical' is not an exhaustive list 
```suggestion
1. **Properties:** agents have properties. A property might be memory, a state, characteristic, or attribute, such as hunger, speed, or health. Properties are discrete, and can include values that are, for example, binary (yes/no), or numerical (e.g. on a 1–100 scale).
```